pvsystemprofiler/latitude_study.py

```diff
@@ -119,13 +131,12 @@ def run(self, data_matrix=('raw', 'filled'),
 
         self.results = results
 
-    def estimate_latitude(self, matrix_id=None, daytime_threshold=0.001,
-                          daylight_method=('sunrise-sunset', 'raw daylight'),
-                          delta_method=('cooper', 'spencer')):
+    def prepare_estimation(self, matrix_id=None, daytime_threshold=0.001,
```

Comment:
call this `prepare_input_data`. have function return a 2-tuple of `(hours_daylight, delta)`